dds2val/requirements/requirements-test.txt

```diff
@@ -3,4 +3,4 @@ pytest-html
 pytest-cov
 pytest-asyncio
 py
-kuksa-client > 0.3
+kuksa-client > 0.4a4
```

Comment:
shouldn't this also be `>=`, assuming that `0.4a4` is the most recent pre-release?